using-the-grid/local-grid-proxy.md

```diff
@@ -0,0 +1,82 @@
+# Creating a grid proxy without LbScripts
+
+Sometimes it is useful to create a grid proxy when LbScripts (and therefore `lhcb-proxy-init`) is unavailable.
+The most commonly use for this is to be able to access data at grid sites using XRootD.
+
+Fortunately a grid proxy can be easily created using [voms](https://italiangrid.github.io/voms/index.html) as follows.
```

Comment:
Maybe worth adding a sentence or two on what voms is?

Reply:
I've added a little, but to be honest I don't really know what it does or how it works 😳